src/sonic-yang-models/yang-models/sonic-subnet-decap.yang

```diff
@@ -0,0 +1,53 @@
+module sonic-subnet-decap {
+
+    yang-version 1.1;
+
+    namespace  "http://github.com/sonic-net/sonic-subnet-decap";
+
+    prefix subnet-decap;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    description "Subnet decap configuration for SONiC OS.";
+
+    revision 2024-12-19 {
+        description "Initial version";
+    }
+
+    container sonic-subnet-decap {
+        container SUBNET_DECAP {
+
+            description "CONFIG_DB subnet decap configuration.";
+
+            list SUBNET_DECAP_LIST {
+
+                key "name";
+
+                leaf name {
+                    type string;
+                    description "Subnet Decap config name.";
+                }
+
+                leaf status {
+					type string {
+						pattern "enable|disable";
+					}
+                    default "disable";
+                    description "Subnet Decap status.";
+                }
+
+                leaf src_ip {
+                    type inet:ipv4-prefix;
+                    description "Subnet decap term source IPv4 prefix.";
+                }
+
+                leaf src_ip_v6 {
```

Comment:
Same above. And this field must be present.

Reply:
Updated, mark this as mandatory.